tribuo-classification/README.md

```diff
@@ -56,6 +56,13 @@ Micro Average            | | | | |
 Macro Average                        | | | | |                                         0.500   |     0.186    |    0.271| 
 Balanced Error Rate        | | | | |                                                   0.500
 
+This sample demonstrates how you can integrate Tribuo and Conclave, to load and train a model. The model is evaluated based
+off the testing data sent by the client. The evaluation result is sent back to the client, and we print it on the client's
+screen. 
+This sample can be extended further to tweak the model parameters, based off the evaluation result. The clients can send
+more training data to better train the model. 
+The clients can further send out some input data, and the model can predict if the input data tumor is malignant or benign.
```

Comment:
Is this actually possible with the current version of the sample? I thought you could only use it for training and evaluating a model.

Haven't looked at this in a while so apologies if I'm mistaken

Reply:
Nope this says that - "This sample can be extended further to...".

Reply:
Oh right, I read this as "This sample can be extended further to tweak the model parameters, based off the evaluation result." ... and then the following sentences you can already do. Maybe it would be clearer to phrase it like

> This sample can be extended in a number of ways:
>    * ...
>    * ...

Reply:
done